### DISCUSSION TITLE
Name error

### DISCUSSION THREAD

**Discussion by allenmoench** [earthlab-education/Earth-Analytics-AY24#624] [Q&A] [answered]
Hello! I'm getting another name error, and not sure how to proceed. It's saying name 'cloud_bits' on line 43 is not defined (even though I defined it on line 24 above...) 

Maybe this is because cloud_bits is defined inside a different function - it's indented within this function:
def process_cloud_mask(cloud_URI, bounds_gdf, bits_to_mask):

![image](https://github.com/user-attachments/assets/1c6d1ca7-f177-4f50-b8a7-b8cf0c7e28ff)

**Comment by allenmoench** [comment, marked as answer]
Never mind, I figured it out! It's an indentation issue again - I'm calling cloud_bits outside the function where I'm defining it.

**Reply by eculler** [reply]
Great observation @allenmoench ! That's a tricky part of getting started with functions and loops in Python.
